javascript.js

```diff
@@ -92,28 +86,21 @@ window.onload = function () {
 
     // OnClick Function
     clickInput = function () {
-        list.onclick = function () {
-            var chosenLetter = (this.innerHTML);
-            this.setAttribute("class", "active");
-            this.onclick = null;
-            
-            for (var x = 0; x < word.length; x++) {
-                if (word[x] === chosenLetter) {
-                    storedLetters[x].innerHTML = chosenLetter;
-                    counter += 1;
-                } 
-            }
-            
-            var i = (word.indexOf(chosenLetter));
-            
-            if (i === -1) {
-                guessesLeft -= 1;
-                textUpdate();
+        var chosenLetter = (this.innerHTML);
+        this.setAttribute("class", "active");
+        this.onclick = null;
+
+        for (var x = 0; x < word.length; x++) {
+            if (word[x] === chosenLetter) {
+                storedLetters[x].innerHTML = chosenLetter;
+                counter += 1;
             } 
-            else {
```

Comment:
textUpdate() was called last in both branches of this if statement, so move it outside.  This also removes the need for an ELSE block